server/services/interventionsService.ts

```diff
@@ -373,6 +373,15 @@ export default class InterventionsService {
     })) as ApprovedActionPlanSummary[]
   }
 
+  async getDeliverySessionAppointments(token: string, referralId: string): Promise<ActionPlanAppointment[]> {
```

Comment:
hm, are we expecting to have initial assessment appointments being returned as well?

I think we should either:
1. Make this generic and have the appointment type defined in the DTO
2. Have it accept an `appointmentType` query param that performs filtering.
3. Change the url to make it more clear that only session-appointments are returned

Reply:
We've changed this to `/delivery-session-appointments` to make it clear that it's just those types being returned.